src/Cli/Utils.cs

```diff
@@ -669,11 +670,20 @@ public static bool TryGetRoleAndOperationFromPermission(IEnumerable<string> perm
             }
             else
             {
+                // For Views and Tables
                 if (parameters is not null && parameters.Any())
                 {
                     _logger.LogError("Tables/Views don't support parameters.");
                     return false;
                 }
+
+                // For Views
+                if (sourceType is SourceType.View && (keyFields is null || !keyFields.Any()))
+                {
+                    _logger.LogError("Key-fields are mandatory for views, but not provided.");
```

Comment:
Because we are enforcing this for all database types, can you update the configuration/CLI documentation noting this behavior? This will add transparency and background context for non PG devs who may not expect keyfields to be required.